src/renderer/components/dock/create-resource.tsx

```diff
@@ -67,6 +90,40 @@ export class CreateResource extends React.Component<Props> {
     return successMessage;
   };
 
+  renderSelectOption(option: string, badge?: string){
+    return this.renderSelectItem(option,"select-template-option", badge );
+  }
+
+  renderSelectGroup(option: string, badge?: string){
+    return this.renderSelectItem(option,"select-template-group", badge );
+  }
+
+  renderSelectItem(option: string, className: string, badge?: string) {
+    return(
+      <div className={cssNames("flex select-template-item", className)}>
+        <span>{option} </span>
```

Comment:
Shouldn't we render the label? or is that what this is.

Also we should only render the space if there is a badge.

Reply:
I throw label here as option: string
https://github.com/lensapp/lens/pull/2327/commits/0b1059c240a86e740a8ff5c4b848874d569159da#diff-6035fe6796405ec877bfcd890b6ee764ac0f3edd60dcd38d2c7f5ba6c7b586edR38
I'll rename it on next commit